server/Core/Entities/Sportsman.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Core.Entities;
+
+public enum Sex
+{
+    M,
+    F
+}
+
+public class Sportsman
+{
+    public int MembershipCardNum { get; set; }
+    public string? SportsCategory { get; set; }
```

Comment:
Perhaps to a separate table?

Reply:
I had plans to do it later. I think strings are enough for now.